cpp/src/io/comp/debrotli.cu

```diff
@@ -74,6 +74,7 @@ namespace io {
 
 #define BREV8(x)    (__brev(x) >> 24u)  // kReverseBits[x]
 
+// @brief Various local scratch arrays
```

Comment:
```suggestion
/// Various local scratch arrays
```
It looks like doxygen suggests using `///` or `//!` for C++-style brief comments (http://www.doxygen.nl/manual/docblocks.html)

Reply:
Updated debrotli. will go through the others...